modules/search/customize-controls/class-excluded-post-types-control.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * A multi-checkbox Customizer control for use with Jetpack Search configuration
+ *
+ * @package jetpack
+ * @since 8.7.0
+ */
+
+/**
+ * Label Control class.
+ */
+class Excluded_Post_Types_Control extends WP_Customize_Control {
+	/**
+	 * Control type.
+	 *
+	 * @since 8.7.0
+	 * @var string
+	 */
+	public $type = 'excluded-post-types';
+
+	/**
+	 * Enqueue styles related to this control.
+	 */
+	public function enqueue() {
```

Comment:
Just a note to check that this enqueuing work ok on WP.com? I think does, but that assumes we place the code in the site-search dir.

Reply:
This won't work as currently coded in WPCOM, I'll investigate a solution for this and the other customize control.

Reply:
I made symlinked Jetpack's `modules/search/customize-controls/` into WPCOM's `site-search` module. This should work as-is if we apply the change correctly for the 8.8 sync.